database/scheduler.go

```diff
@@ -0,0 +1,119 @@
+package database
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	_ "github.com/mattn/go-sqlite3"
+	"log"
+	"time"
+
+	"TODOGo/config"
+	"TODOGo/schemas"
+)
+
+// AddTaskToDB вставляет новую задачу в базу данных.
+func AddTaskToDB(task schemas.Table) (uint64, error) {
+	query := "INSERT INTO scheduler (date, title, comment, repeat) VALUES (?, ?, ?, ?)"
+	res, err := config.DB.Exec(query, task.Date, task.Title, task.Comment, task.Repeat)
+	if err != nil {
+		return 0, fmt.Errorf("ошибка вставки задачи: %v", err)
```

Comment:
При выводе `%v`, а при оборачивании `%w`(wrap) 
```suggestion
		return 0, fmt.Errorf("ошибка вставки задачи: %w", err)
```